packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/FrameCarousel.js

```diff
@@ -16,11 +16,7 @@ export const StyledControlButton = styled(Button)`
     fill: #FFFFFF;
     stroke: #FFFFFF;
   }
-  &:focus {
-    background: #7fcbce;
-    box-shadow: none;
-  }
-  &:hover {
+  &:focus, &:hover {
     background: #7fcbce;
     box-shadow: none;
```

Comment:
Out of scope of this PR, but a lot of these styles can be addressed via props or theme I think and the way the icon is applied to the Button changed. Happy to discuss in person these improvements.

Reply:
I'd like to refactor the styling in #1525